README.md

```diff
@@ -0,0 +1,139 @@
+<a href="https://livekit.io/">
+  <img src="./.github/assets/livekit-mark.png" alt="LiveKit logo" width="100" height="100">
+</a>
+
+# LiveKit Templates Index
+
+<p>
+  <a href="https://cloud.livekit.io/projects/p_/sandbox"><strong>Deploy a sandbox app</strong></a>
+  •
+  <a href="https://docs.livekit.io">LiveKit Docs</a>
+  •
+  <a href="https://livekit.io/cloud">LiveKit Cloud</a>
+  •
+  <a href="https://blog.livekit.io/">Blog</a>
+</p>
+
+This repository contains a collection of templates for the LiveKit platform. Each template is designed to be used as a starting point for building an application on LiveKit, as well as to work with the [LiveKit Sandbox](https://docs.livekit.io/cloud/home/sandbox) feature to rapidly develop and share app prototypes. 
+
+## Using a template
+
+All templates in this index can be bootstrapped using the [LiveKit CLI](https://docs.livekit.io/cloud/home/cli). Running the following command will allow you to choose and initialize a template:
+
+```bash
+lk app create
+```
+
+After choosing a template and name for your project, the CLI will execute the following steps:
+
+1. Clone the git repository associated with the template to your `/tmp` directory or equivalent
+2. De-git the repository so you don't include the template's history in your project
+3. Copy the `.env.example` file to `.env.local`
+    - Automatically filling in any `LIVEKIT_` environment variables that are recognized
+    - Prompting you to fill in any additional variables that are not recognized, such as required API keys
+4. Move the directory from `/tmp` to the location you specified
+5. Run the `post_create` task from the `taskfile.yaml` and print any user setup instructions
+
+At this point, you're ready to start developing your application! Some templates work best with others -- for example, an AI agent template may pair well with a voice assistant frontend template. It all depends on what you want to build. The beauty of LiveKit is that the primitives are interchangeable!
```

Comment:
The en dash should be an em dash.

And personally, I would remove the exclamation points. Or at least limit it to only one per paragraph!

```suggestion
At this point, you're ready to start developing your application! Some templates work best with others --- for example, an AI agent template may pair well with a voice assistant frontend template. It all depends on what you want to build. The beauty of LiveKit is that the primitives are interchangeable!
```